### DISCUSSION TITLE
I want my workflow to run once every day at 9 am. How can I do it?

### DISCUSSION THREAD

**Discussion by weiweimaodetuzi** [langgenius/dify#11781] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I created a workflow on Dify. Its function is to send the returned data to the group message after requesting the HTTP interface. Now I want this workflow to run once every day at 9 am. How can I do it?

**Comment by ianZzzzzz** [comment]
interested👀

**Reply by Yang-yyf** [reply]
The easiest way to meet your needs is to write a crontab service on your Linux server and then call your Dify API endpoint every morning at 9 am.
